core/coretypes/src/ratio_impl.cpp

```diff
@@ -10,10 +10,10 @@ BEGIN_NAMESPACE_OPENDAQ
 
 RatioImpl::RatioImpl(Int numerator, Int denominator)
     : GenericStructImpl<daq::IRatio, daq::IStruct, daq::IConvertible, daq::IComparable>(
-          RatioStructType(), Dict<IString, IInteger>({{"numerator", numerator}, {"denominator", denominator}}))
+          RatioStructType(), Dict<IString, IInteger>({{"Numerator", numerator}, {"Denominator", denominator}}))
```

Comment:
It'd be good to have a check for everything that has a struct type whether deserialization with old struct types works. Since here we're basically redefining all struct types to be PascalCase instead of camelCase.

I'd just include a test suite that has a set of serialized object strings of structs as they were before the change, then the test should try to deserialize them with the new IDs.

Reply:
I believe this shouldn't be a problem. For example, a Ratio may serialize into `{"__type":"Ratio","num":1,"den":2}`, which is the same as before, because `SerializeId()` is the same (`Ratio`), and so are the keys (`num` and `den`). Am I wrong?

Reply:
It shouldn't be an issue as long as these StructTypes are not serialized. The only result will likely be that there will be a duplicate entry in the TypeManager of these struct types on the native client side - one entry will be camelCase the other PascalCase. It shouldn't cause issues, however.